Library/Formula/apcupsd.rb

```diff
@@ -0,0 +1,106 @@
+class Apcupsd < Formula
+  desc "Daemon for controlling APC UPSes"
+  homepage "http://www.apcupsd.org"
+  url "https://downloads.sourceforge.net/project/apcupsd/apcupsd%20-%20Stable/3.14.13/apcupsd-3.14.13.tar.gz"
+  sha256 "57ecbde01d0448bf8c4dbfe0ad016724ae66ab98adf2de955bf2be553c5d03f9"
+
+  option "with-usb", "Build with support for USB UPSes."
+
+  depends_on "gd"
+  depends_on "libusb-compat"
```

Comment:
Is this only required if using usb support? If so, let's make it `=> :optional` and rename the `with-usb` option to `with-libusb-compat`